jms/src/main/scala/akka/stream/alpakka/jms/Jms.scala

```diff
@@ -63,7 +71,15 @@ final case class JmsConsumerSettings(connectionFactory: ConnectionFactory,
   def withSessionCount(count: Int): JmsConsumerSettings = copy(sessionCount = count)
   def withBufferSize(size: Int): JmsConsumerSettings = copy(bufferSize = size)
   def withQueue(name: String): JmsConsumerSettings = copy(destination = Some(Queue(name)))
+  def withQueue(
+      name: String,
+      createDestination: (String) => (jms.Session) => jms.Destination = Destination.createQueue
```

Comment:
The `withXyz` methods are meant for use from Java, this signature will put off Java users.

Reply:
Thanks for the alert?

Any suggestion how to please Java user ?